R/MsBackendCached.R

```diff
@@ -0,0 +1,614 @@
+setClassUnion("characterOrInteger", c("character", "integer"))
+
+#' @title Base MsBackend class providing data caching mechanism
+#'
+#' @aliases MsBackendCached-class MsBackendCached
+#'
+#' @description
+#'
+#' The `MsBackendCached` class is a rudimentary implementation of the
+#' [MsBackend] providing a simple mechanism to cache spectra data locally. This
+#' class is thought to be used as a base class for other `MsBackend`
+#' implementations to reuse its caching mechanism and avoid having to
+#' re-implement commonly used methods. This class is thus not thought to be
+#' used directly by a user.
+#'
+#' The provided caching mechanism by this class allows `MsBackend` instances
```

Comment:
The `MsBackendCached` caching mechanism allows ...

Reply:
Thanks